### PR TITLE
Updated Jupyter notebooks for exchange family 

### PR DESCRIPTION
**Pull request by cmickeyb**
Significant update for the Jupyter notebooks in the exchange contract family. Notebooks added for creating and importing orders, for interacting with an issuer as an account holder, and for kick starting the factories there is a new index file.

To make all this work, this update adds an experimental file format for sharing contract information. The file uses the python zip library (distributed with python) to package the PDO files and the context files. 

Updated from earlier draft PR.

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -5,9 +5,28 @@ https://creativecommons.org/licenses/by/4.0/
...
+
+```
+$ ${PDO_INSTALL_ROOT}/bin/pip3 install ${EXCHANGE_PACKAGE} ${DIGITAL_ASSET_PACKAGE}
+$ ${PDO_INSTALL_ROOT}/bin/pdo-install-plugin-resources --module pdo.exchange --family exchange
+$ ${PDO_INSTALL_ROOT}/bin/pdo-install-plugin-resources --module pdo.digital_asset --family digital_asset
+```
+
+Note that the build process described below will create the Python packages in
+the directory `${PDO_CONTRACTS_ROOT}/build/dist`.
+
```
Comment (prakashngit) [inline docs/install.md:28]: the whole section on Install Pre-Built Packages is confusing. 

1. where do we get the pre-built packages from? 
2. If we have already done make/make install, why are these commands needed?

File: exchange-contract/docs/notebooks/README.md [hunk 2]
```diff
@@ -87,23 +87,7 @@ PDO initialization handles configuration of the PDO modules, this
...
-
-Templates provide the skeletons for the various contract objects. In
-general, you would not open a template directly. Instead, use on the
-factories to create the token objects from the templates.
-
-* `issuer.ipynb` -- template for an asset issuer
-* `token.ipynb` -- template for a token
-* `token-issuer.ipynb` -- template for a token issuer
+The easiest way to get started is to use the Jupyter file browser to
+open `index.ipynb`. Render all of the markdown cells.
```
Comment (prakashngit) [inline exchange-contract/docs/notebooks/README.md:93]: Please add instructions to copy index.ipynb to $PDO_HOME/notebooks folder

File: exchange-contract/docs/notebooks/index.ipynb [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bc2fe555-a933-42e4-ad25-421e56baaac5",
+   "metadata": {},
+   "source": [
+    "### Create a personal configuration file\n",
+    "\n",
+    "The default PDO client configuration file is in `${PDO_INSTALL_ROOT}/opt/pdo/etc/pcontract.toml`. This file contains a basic configuration that should suffice for testing on a single host. \n",
+    "\n",
```
Comment (prakashngit) [inline exchange-contract/docs/notebooks/index.ipynb:53]: not sure i understand this. you already above gave the command to create the groups file based on default configuration. are you saying that "a future notebook to override the default config, and create the groups file based on custom config" ?

**Review by prakashngit — COMMENTED** [review]
issuer notebook execution fails while creating contracts. asset contract gets created, but the vetting contact creation fails with FileNotFoundError: [Errno 2] No such file or directory: private-data-objects/build/_dev/opt/pdo/etc/context/tmptqbtilzk

**Comment by cmickeyb** [comment]
Replaced by #29

**Pull request closed without merging**